tests/test_lib_audit.py

```diff
@@ -291,3 +293,42 @@ def test_10_check_tokentype(self):
         self.assertEqual(audit_log.total, 1)
         # The tokentype was actually written as token_type
         self.assertEqual(audit_log.auditdata[0].get("token_type"), "spass")
+
+    def test_20_logger_audit(self):
+        a = LoggerAudit()
+        a.log({"action": "something"})
+        a.finalize_log()
+        r = a.search({"action": "something"})
+        # This is a non readable audit, so we got nothing
+        self.assertEqual(r.auditdata, [])
+        self.assertEqual(r.total, 0)
```

Comment:
i think in the long run we should also check that a given logger audit configuration does work and sends the logs to a file.

Reply:
I do not understand your comment. 
Do you mean check it in the running system or add a test? 
If you think there should be done "in the future", we should open an issue for this.

Reply:
Yes, i was thinking about a test with a given log config. I'll try to come up with something.